schema.sql

```diff
@@ -24,4 +24,26 @@ CREATE TABLE species (
 
 ALTER TABLE animals
 ADD COLUMN species_id INTEGER,
-ADD COLUMN owner_id INTEGER;
\ No newline at end of file
+ADD COLUMN owner_id INTEGER;
+
+CREATE TABLE vets (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT,
```

Comment:
[OPTIONAL]
- [ ] I would suggest you use `VARCHAR(255)` for example for your name since `text` is dedicated to long paragraph